onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -1467,6 +1467,41 @@ TEST_F(GraphTransformationTests, ReshapeFusionConcatSubgraph) {
   }
 }
 
+TEST_F(GraphTransformationTests, ReshapeFusionWithSlice1) {
+  auto model_uri = MODEL_FOLDER "fusion/reshape_fusion_with_slice1.onnx";
```

Comment:
Should test\testdata\transform\fusion\reshape_fusion_gen.py be updated to generate this file?

Reply:
Otherwise the change looks good to me.